From a7387816379b492946dcea4bb7d1dd3595c147b5 Mon Sep 17 00:00:00 2001
From: chettoy <chettoy@with.moe>
Subject: [PATCH 1/2] Fix "multiple definition of `cont_box_drop'"

---
 memflow-ffi/memflow.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/memflow-ffi/memflow.h b/memflow-ffi/memflow.h
index 19919ebb..95be1624 100644
--- a/memflow-ffi/memflow.h
+++ b/memflow-ffi/memflow.h
@@ -2383,10 +2383,10 @@ static CArc_c_void ctx_arc_clone(CArc_c_void *self) {
     return ret;
 }
 
-void ctx_arc_drop(CArc_c_void *self) {
+inline void ctx_arc_drop(CArc_c_void *self) {
     if (self->drop_fn && self->instance) self->drop_fn(self->instance);
 }
-void cont_box_drop(CBox_c_void *self) {
+inline void cont_box_drop(CBox_c_void *self) {
     if (self->drop_fn && self->instance) self->drop_fn(self->instance);
 }
 
-- 
2.43.0

